kratos/python_scripts/gid_output_process.py

```diff
@@ -645,3 +653,48 @@ def __remove_list_files(self):
                         os.remove(f)
                     except WindowsError:
                         pass
+                    
+    #
+    def __remove_post_results_files(self, step_label):
```

Comment:
Why you are using the double underline "__" for this method? The naming convention of Python [here](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) says that it would do the name mangling to the class name which I'm not sure is what you intended.